update_all

```diff
@@ -9,19 +9,20 @@ fi
 
 # arb1
 yarn fullList --l2NetworkID 42161
-yarn arbify --l2NetworkID 42161 --tokenList https://gateway.ipfs.io/ipns/tokens.uniswap.org
-yarn arbify --l2NetworkID 42161 --tokenList https://www.gemini.com/uniswap/manifest.json
-yarn arbify --l2NetworkID 42161 --tokenList https://api.coinmarketcap.com/data-api/v3/uniswap/all.json
+yarn arbify --l2NetworkID 42161 --tokenList https://gateway.ipfs.io/ipns/tokens.uniswap.org --newArbifiedList ./src/ArbTokenLists/arbed_uniswap_labs.json
```

Comment:
Can we update the CI too, so we generate them with the proper name?

Reply:
Sure